code/datums/components/food_stuff.dm

```diff
@@ -0,0 +1,238 @@
+				if (/obj/item/skull)
+					boutput(L, "<span class='alert'>Ugh. Nothing but bone.</span>")
+					return
+				if (/obj/item/skull/strange)
+					boutput(L, "<span class='alert'>Ugh. Nothing but bone. Pretty spooky though.</span>")
+					return
+				if (/obj/item/skull/peculiar)
+					playsound(L, "sound/misc/meat_plop.ogg", 100, 1)
+					L.visible_message("<span class='alert'>[M] vomits <i>everywhere</i>.</span>", "<span class='alert'><b>UUAAAUGGHHH...</b> The wizard's skull was cursed.</span>")
+					L.emote("scream")
+					L.changeStatus("paralysis", 10 SECONDS)
+					L.abilityHolder.deductPoints(10, target_abilityholder)
+					for (var/turf/T in range(L, rand(1, 3)))
+						if (prob(20))
+							make_cleanable( /obj/decal/cleanable/greenpuke,T)
+						else
+							make_cleanable( /obj/decal/cleanable/vomit,T)
+					return
+				if (/obj/item/skull/noface)
+					L.vomit()
+					L.abilityHolder.deductPoints(2, target_abilityholder)
+					boutput(L, "<span class='alert'>Eugh, that skull was so sickeningly <i>sweet</i>.</span>")
+					return
+				if (/obj/item/skull/crystal)
+					add_these_points = 10
+					L.emote("scream")
+					L.changeStatus("paralysis", 30 SECONDS)
+					boutput(L, "<span style=\"color: red; font-family: Fixedsys, monospace; text-align: center; vertical-align: top; -dm-text-outline: 1 black;\">YOU HAVE TASTED ALL THAT THE UNIVERSE HAS TO HOLD OF EVIL.</span>")
+					SPAWN_DBG(rand(5 SECONDS, 10 SECONDS))
+						L.vomit()
+				if (/obj/item/skull/gold)
+					L.emote("scream")
+					L.changeStatus("weakened", 10 SECONDS)
+					boutput(L, "<span class='alert'>The moment it reaches your stomach, the skull headbutts you right in the solar plexus! Oof...</span>")
+					SPAWN_DBG(rand(1 SECOND, 3 SECONDS))
+						L.emote(pick("groan", "moan"))
+					return
+				if (/obj/item/skull/odd)
+					add_these_points = 4
+					boutput(L, "<span class='notice'>Delicious, with an oddly familiar aftertaste.</span>")
+					boutput(L, "<span class='notice'>You feel a slight wriggling in your gut.</span>")
+					SPAWN_DBG(rand(3 SECONDS, 10 SECONDS))
+						boutput(L, "<span class='notice'>The wriggling passes.</span>")
+						L.emote("fart")
+				if (/obj/item/organ/brain)
+					boutput(L, "<span class='notice'>Delicious! The creamy, savory taste of [I] leaves you with a big dumb grin.</span>")
+				if (/obj/item/organ/brain/synth)
+					boutput(L, "<span class='notice'>Not quite as tasty as a <i>real</i> brain, but it tastes a lot less... kuru-y.</span>")
+				if (/obj/item/organ/chest)
+					boutput(L, "<span class='notice'>Bland, but there was a lot of it.</span>")
+				if (/obj/item/organ/heart)
+					boutput(L, "<span class='notice'>Full of iron!</span>")
+				if (/obj/item/organ/heart/synth)
+					boutput(L, "<span class='notice'>Full of pharosium!</span>")
+				if (/obj/item/organ/heart/flock)
+					boutput(L, "<span class='notice'>Full of iron...?</span>")
```

Comment:
i would say gnesis or blue goop, but vOv

Reply:
Changed to taste like adjective chicken.